clients/spark-3.2-extensions/build.gradle.kts

```diff
@@ -60,7 +60,7 @@ dependencies {
   testImplementation("org.junit.jupiter:junit-jupiter-params")
   testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine")
 
-  nessieQuarkusServer("org.projectnessie:nessie-quarkus:$clientNessieVersion:runner")
+  nessieQuarkusServer(project(":nessie-quarkus", "quarkusRunner"))
```

Comment:
help me understand (ideally in commit msg) why we only need to run against the in-tree version now.

do we not lose important test coverage by no longer running against `clientNessieVersion` ?
i thought due to the cyclic dependency we need to test the spark extensions together with the nessie version that was used in the iceberg release we are depending on?

do we maybe we need to run tests against both?
potentially with version specific code in the tests.

Reply:
This one changes the _server_, not the client version. The Nessie _client_ version must be the same as the one used by Iceberg.

Reply:
ah ok i see

so in the past we ran:
the in-tree extensions with the old client against the old server.
and now we run:
the in-tree extensions with the old client against the in-tree server?

makes sense then